cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/inmemory/InMemoryMapReduceContextBuilder.java

```diff
@@ -86,7 +88,13 @@ private Injector createInMemoryModules() {
       new LoggingModules().getInMemoryModules(),
       new StreamAdminModules().getInMemoryModules(),
       new ExploreClientModule(),
-      new NotificationFeedServiceRuntimeModule().getInMemoryModules()
+      new NotificationFeedServiceRuntimeModule().getInMemoryModules(),
+      new AbstractModule() {
+        @Override
+        protected void configure() {
+          bind(Store.class).to(DefaultStore.class);
```

Comment:
We need it here as well, because ProgramRunner is also created by this injector on  line 84 above. right?